src/coreclr/scripts/jitformat.py

```diff
@@ -208,26 +208,23 @@ def main(argv):
         logging.info("Creating patch file {}".format(patchFilePath))
         jitSrcPath = os.path.join(runtime, "src", "coreclr", "jit")
         patchFile = open(patchFilePath, "w")
-        proc = subprocess.Popen(["git", "diff", "--patch", "-U20", "--", jitSrcPath], env=my_env, stdout=patchFile)
+        proc = subprocess.Popen(["git", "diff", "--patch", "-U20", "--", jitSrcPath], cwd=runtime, env=my_env, stdout=patchFile)
```

Comment:
Should we make this optional (e.g. based on a command-line flag)?

Reply:
This change is technically required to make this work, and it has no difference if your current directory is in the `runtime` folder or a subfolder.

Reply:
Ah, sorry I thought it is a string "runtime". 😅